### DISCUSSION TITLE
OneDrive LogFile not showing

### DISCUSSION THREAD

**Discussion by AdziePahlevi** [abraunegg/onedrive#1854] [Q&A] [answered]
Hello Expert, 

thanks a lot for developing the apps, it is really works. 
I have a problem where the default log to the specified directory on my onedrive config were not working 

I was following the following case:
https://github.com/abraunegg/onedrive/discussions/1772

and following the guidelines in here:
https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#client-activity-log

![image](https://user-images.githubusercontent.com/19539988/155070336-cee15987-d05c-47ce-b20b-8288ad4281e5.png)

but the file still not there. below is my onedrive config

![image](https://user-images.githubusercontent.com/19539988/155070493-6f5ea0ed-94b7-436b-8cd7-6de4f35e356c.png)

![image](https://user-images.githubusercontent.com/19539988/155070562-5abde4ce-91d6-43f2-a5fd-9f367054364e.png)

I confirmed all SSH as been logged out and I was thinking to reboot my PC, is it worth to try? since this PC is also running my home media and pfsense and I rarely reboot it.

**Comment by abraunegg** [comment, marked as answer]
@AdziePahlevi 
> I have a problem where the default log to the specified directory on my onedrive config were not working

Based on your configuration file - you have not enabled logging as it is still set to "false" ... you need to update this option.

**Reply by AdziePahlevi** [reply]
> @AdziePahlevi
> 
> > I have a problem where the default log to the specified directory on my onedrive config were not working
> 
> Based on your configuration file - you have not enabled logging as it is still set to "false" ... you need to update this option.

ah.. I missed that.. let me change the config... thanks a lot.

**Comment by AdziePahlevi** [comment]
already change enable logging to TRUE

![image](https://user-images.githubusercontent.com/19539988/155093093-4dd2677c-94bc-425b-b21b-0a185d6e7b96.png)

config after I change the configruration

![image](https://user-images.githubusercontent.com/19539988/155093145-5dccaa7c-5e60-40d3-8450-9311a47d93c4.png)

I tried to run dry-run, however I dont see any flag related to log

![image](https://user-images.githubusercontent.com/19539988/155093209-1c12f44b-e247-4cbe-aeed-993bf65502ec.png)


and the log file still not generated

![image](https://user-images.githubusercontent.com/19539988/155093324-8108e760-6acc-4c69-a4aa-cfe2bc139cc1.png)

this is snapshot of my user groups

![image](https://user-images.githubusercontent.com/19539988/155093613-538a5b86-9e63-48ce-bbf6-9d1f70942d7c.png)

any other advices

**Comment by abraunegg** [comment]
@AdziePahlevi 
> and the log file still not generated

This is now because your 'modified' configuration file is located here: `/home/adzie/.config/onedrive/accounts/H3I/config`

But when you are running the application via your command: `onedrive --dry-run --verbose --monitor` - you are using the configuration file `/home/adzie/.config/onedrive/config` .. which, if it does not exist, application defaults will be used ... this is *why* this message is displayed: `No user or system config file found, using application defaults`

So how do you fix this situation:

**Option 1:** Specify your configuration using the `--confdir` option
**Option 2:** Fix your 'default' configuration file

**Reply by AdziePahlevi** [reply]
> 

excellent, basically I just copied the conf file from `/home/adzie/.config/onedrive/accounts/H3I/config` to `/home/adzie/.config/onedrive/`

then voila it works like a charm... thanks a lot @abraunegg
